includes/donors/actions.php

```diff
@@ -86,3 +90,28 @@ function __give_remove_donor_donation_comment( $donation_id ) {
 }
 
 add_action( 'give_payment_deleted', '__give_remove_donor_donation_comment', 10 );
+
+
+/**
+ * Update anonymous donation for legacy gateways
+ *
+ * @unreleased
+ *
+ * @param $donationId
+ *
+ */
+function updateAnonymousDonationForLegacyGateways($donationId)
+{
+    $gatewayId = give_get_meta($donationId,'_give_payment_gateway');
```

Comment:
@jonwaldstein Can we set the third param to `true`. That means the function will return a string value.